force_x: 0.8
force_y: 0.5
force_z: 0.5
